src/main/java/org/kiwiproject/dropwizard/error/config/CleanupConfig.java

```diff
@@ -24,12 +24,32 @@
     }
 
     /**
-     * The strategy to use for what to cleanup. Defaults to {@link CleanupStrategy#RESOLVED_ONLY}
+     * The strategy to use for what to cleanup. Defaults to {@link CleanupStrategy#ALL_ERRORS}
      */
-    private CleanupStrategy cleanupStrategy = CleanupStrategy.RESOLVED_ONLY;
+    private CleanupStrategy cleanupStrategy = CleanupStrategy.ALL_ERRORS;
 
     /**
-     * The duration of time that an error will live before being deleted.
+     * The duration of time that a resolved error will live before being deleted.
      */
-    private Duration applicationErrorExpiration = Duration.days(30);
+    private Duration resolvedErrorExpiration = Duration.days(14);
+
+    /**
+     * The duration of time that an unresolved error will live before being deleted.
+     */
+    private Duration unresolvedErrorException = Duration.days(60);
+
+    /**
+     * The name to give the scheduled job for cleaning up errors. Defaults to {@code Cleanup-Job-%d}
```

Comment:
Maybe add something to the end so it tells you what the thread names will looks like, w.g.

> The name to give the scheduled job for cleaning up errors. Defaults to {@code Cleanup-Job-%d}
which will result in thread names like {@code Cleanup-Job-1}